install/kubernetes/helm/istio/values.yaml

```diff
@@ -353,22 +353,18 @@ prometheus:
   image:
     repository: docker.io/prom/prometheus
     tag: latest
-  ingress:
+  ingressgateway:
+    # Warning: Prometheus does not support authn/authz. Exposing
+    # Prometheus via ingressgateway in this method will allow
+    # unauthenticated access to the Prometheus UI.
     enabled: false
-    # Used to create an Ingress record.
-    #hosts:
-    #  - prometheus.local
-    annotations:
-      # kubernetes.io/ingress.class: nginx
-      # kubernetes.io/tls-acme: "true"
-    tls:
-      # Secrets must be manually created in the namespace.
-      # - secretName: prometheus-tls
-      #   hosts:
-      #     - prometheus.local
+    host: "prometheus.local"
   service:
     annotations: {}
     nodePort:
+      # Do not enable both ingressgateway and nodePort at the same
```

Comment:
Please add an option to specify which gateway to use ( the label in gateway should be customizable ). 
Once we do this, user can create a Gateway of type node port, and we can remove the node port option from prom, grafana, etc. Instead users will point them to the gateway-nodeport